src/unread/unreadModel.js

```diff
@@ -271,7 +156,8 @@ function streamsReducer(
     }
 
     case EVENT_MESSAGE_DELETE:
-      return deleteMessages(state, action.messageIds, globalState.messages);
+      // TODO optimize by using `state.messages` to look up directly
```

Comment:
IIUC "by using `state.messages`" is no longer accurate (here and below), but we do have #4684 to track the planned solution that doesn't use `state.messages`, so this might be fine as long as we don't lose track of #4684.

Reply:
Ah indeed, good catch. Updated the comments.